config/default.yml

```diff
@@ -202,6 +202,11 @@ Rails/EnvironmentComparison:
   Enabled: true
   VersionAdded: '0.52'
 
+Rails/Exclude:
```

Comment:
I feel this name might be too general, especially if other cops beginning with the word 'Exclude` could be added in future. But I don't have a good suggestion for an alternative.

Reply:
Yes, I had the same feeling from the beginning, but saw multiple cops with similar names (like `Pick` or `IndexBy`) 😄 

How about `ActiveSupportExclude` or `EnumerableExclude` or `NotInclude`?

Reply:
I'd vote for `NotInclude`, but let's try get some other opinions.

Reply:
`NegateInclude` sounds good.